xcdat/bounds.py

```diff
@@ -428,239 +467,238 @@ def _create_bounds(self, axis: CFAxisKey, coord_var: xr.DataArray) -> xr.DataArr
 
         return bounds
 
-    def _validate_axis_arg(self, axis: CFAxisKey):
-        cf_axis_keys = CF_ATTR_MAP.keys()
+    def _create_time_bounds(
+        self,
+        time: xr.DataArray,
+        freq: Optional[Literal["year", "month", "day", "hour"]] = None,
+        daily_freq: Optional[Literal[1, 2, 3, 4, 6, 8, 12, 24]] = None,
+    ) -> xr.DataArray:
+        """Creates time bounds for each timestep of the time coordinate axis.
 
-        if axis not in cf_axis_keys:
-            keys = ", ".join(f"'{key}'" for key in cf_axis_keys)
-            raise ValueError(
-                f"Incorrect 'axis' argument value. Supported values include {keys}."
-            )
+        This function uses ``_infer_freq`` to determine the temporal resolution and
+        then calls ``get_yearly_time_bounds``, ``get_monthly_time_bounds``, or
+        ``get_daily_time_bounds`` as appropriate. See the docstrings for these
+        functions for more details.
```

Comment:
```suggestion
```

Reply:
I don't think we need this documentation anymore since `_create_time_bounds()` is a private method and the "Parameters" docstrings include this info.